### PR TITLE
feat: leadOneAndFour for 1366px

### PR DESCRIPTION
**Pull request by apostolnikov**
[Story](https://nidigitalsolutions.jira.com/browse/REPLAT-8553)

As discussed with the designers, if there is no star or flags we should fill the left whitespace with teaser text for TileAD(right four slices in the slice)

Section fragment is updated by adding summary300 and teaser300 on LeadOneAndFourSlice.

image

With star: 
image

### CODE REVIEW

File: packages/edition-slices/src/slices/leadoneandfour/index.js [hunk 1]
```diff
@@ -73,8 +93,8 @@ class LeadOneAndFour extends Component {
   render() {
     return (
       <ResponsiveSlice
-        renderSmall={this.renderSmall}
         renderMedium={this.renderMedium}
+        renderSmall={this.renderSmall}
```
Comment (infreezer) [inline packages/edition-slices/src/slices/leadoneandfour/index.js]: I know it's very small thing. But I have to ask.
Is there a reason to change the order of the props?

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js [hunk 2]
```diff
@@ -5,11 +5,16 @@ import {
   editionBreakpoints
 } from "@times-components/styleguide";
 
-const paddingVertical = {
+const paddingVerticalResolver = {
   [editionBreakpoints.small]: spacing(3),
-  [editionBreakpoints.medium]: spacing(3),
+  [editionBreakpoints.medium]: spacing(8),
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-ac/styles/index.js]: Nice one.

File: packages/edition-slices/src/slices/leadoneandfour/index.js (hunk 1 above)
Comment (apostolnikov) [reply packages/edition-slices/src/slices/leadoneandfour/index.js]: No idea how the order was changed or if it was like that before... anyway going to change it 👍

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js [hunk 3]
```diff
@@ -34,6 +39,6 @@ export default breakpoint => ({
     justifyContent: "center",
     backgroundColor: colours.functional.border,
     paddingHorizontal: spacing(4),
-    paddingVertical: paddingVertical[breakpoint] || spacing(3)
+    paddingVertical: paddingVerticalResolver[breakpoint] || spacing(3)
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-ac/styles/index.js]: Do we need the default value?

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js [hunk 4]
```diff
@@ -5,11 +5,16 @@ import {
...
-  [editionBreakpoints.medium]: spacing(3),
+  [editionBreakpoints.medium]: spacing(8),
   [editionBreakpoints.wide]: spacing(6),
-  [editionBreakpoints.huge]: spacing(6)
+  [editionBreakpoints.huge]: spacing(14)
+};
+
+const fontSizeResolver = {
+  [editionBreakpoints.wide]: 30,
+  [editionBreakpoints.huge]: 35
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-ac/styles/index.js]: The font size for medium breakpoint is missing.
Please check it.

File: packages/edition-slices/src/tiles/tile-ad/styles/index.js [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
+  fonts,
+  spacing,
+  editionBreakpoints
+} from "@times-components/styleguide";
 
-const styles = {
+const fontSizeResolver = {
+  [editionBreakpoints.wide]: 20,
+  [editionBreakpoints.huge]: 22
+};
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-ad/styles/index.js]: Please check the slice on medium brekpoint.

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js (hunk 3 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-ac/styles/index.js]: IMO it's not needed and in my PR I prefer to remove it

Comment (apostolnikov) [reply packages/edition-slices/src/tiles/tile-ac/styles/index.js]: Yes, we do need default value. Seems that the breakpoint is not passed on time for the snap and will get undefined value in the test afterward. Thus, will leave it like that for now.

Comment (infreezer) [reply packages/edition-slices/src/tiles/tile-ac/styles/index.js]: The breakpoint is not passed at all.
Let's leave it like it is for now.
When [this PR](https://github.com/newsuk/times-components/pull/2288) is merged the snapshot will be fixed.

File: packages/edition-slices/__tests__/android/__snapshots__/tiles-with-style.test.js.snap [hunk 6]
```diff
@@ -1841,9 +1823,9 @@ exports[`35. tile ac 1`] = `
         Object {
           "color": "#1D1D1B",
           "fontFamily": "TimesModern-Bold",
-          "fontSize": 30,
+          "fontSize": undefined,
```
Comment (infreezer) [inline packages/edition-slices/__tests__/android/__snapshots__/tiles-with-style.test.js.snap]: Ignore this. No breakpoint is passed when testing the tile.
Will be fixed soon in [this PR](https://github.com/newsuk/times-components/pull/2288).

**Pull request merged**